server/api.js

```diff
@@ -313,8 +313,50 @@ router.delete("/laptop_assignment/:assignmentId", async (request, response) => {
 				"(SELECT * FROM laptop_donation d WHERE d.id > $1 and (SELECT COUNT(*) FROM laptop_assignment a WHERE a.laptop_donation_id = d.id) < d.number_of_laptops ORDER BY d.id LIMIT 1)",
 				[queryResult.rows[0].laptop_donation_id]
 			);
+			const unAssignedRequests = await db.query(
+				"SELECT id, uuid, firstname FROM laptop_request  WHERE id > $1 and id NOT IN (SELECT laptop_request_id FROM laptop_assignment) and laptop_request_status != 'CANCELLED' ",
```

Comment:
I'd really like to see an explicit `ORDER BY` clause here (like in the `nextDonation` query) - while de facto a database engine is likely to fetch rows out oldest first, that isn't guaranteed to be the case, and when the order of a result set is important it's always good to explicitly think about that and include it within the query.

Reply:
I am now explicitly ordering the data from the query

Reply:
I can still see the query

```sql
SELECT id, uuid, firstname FROM laptop_request  WHERE id > $1 and id NOT IN (SELECT laptop_request_id FROM laptop_assignment) and laptop_request_status != 'CANCELLED'
```

which doesn't have an `ORDER BY` clause. To be clear, not a blocker for merging!